proto/protocolapis/astria/protocol/transactions/v1alpha1/types.proto

```diff
@@ -181,3 +182,21 @@ message BridgeLockAction {
   // will receive the bridged funds
   string destination_chain_address = 5;
 }
+
+// `BridgeUnlockAction` represents a transaction that transfers
+// funds from a bridge account to a sequencer account.
+//
+// It's the same as a `TransferAction` but with the added
+// `memo` field.
+message BridgeUnlockAction {
+  // the address of the bridge account to transfer to
+  astria.primitive.v1.Address to = 1;
+  // the amount to transfer
+  astria.primitive.v1.Uint128 amount = 2;
+  // the asset to be transferred
+  bytes asset_id = 3;
```

Comment:
don't think we need this field - since bridge accounts only allow one asset to be bridged, we know using `BridgeUnlockAction` must also withdraw that same asset